comps/imagegen/triton/Dockerfile.ubuntu

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 HabanaLabs, Ltd0
```

Comment:
Is this Dockerfile borrowed from `HabanaLabs`? If yes, why not use their published container and add our layer to it?

Reply:
Let me check with the engineer that helped me with  this base layer - I think I remember something about them only shipping the dockerfile, and not distributing the built image...